sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessageSerializer.java

```diff
@@ -225,6 +234,62 @@ public int getSize(Message amqpMessage) {
         return amqpMessage;
     }
 
+    private Map<String, Object> convertToDescribedType(Map<String, Object> propertiesValue) {
+        for (Map.Entry<String, Object> entry : propertiesValue.entrySet()) {
+            Object value = entry.getValue();
+            if (value instanceof URI) {
+                propertiesValue.put(entry.getKey(), new ServiceBusDescribedType(ServiceBusConstants.URI_SYMBOL, ((URI) value).toString()));
+            } else if (value instanceof OffsetDateTime) {
+                propertiesValue.put(entry.getKey(), new ServiceBusDescribedType(ServiceBusConstants.OFFSETDATETIME_SYMBOL, convertToTickTime((OffsetDateTime) value)));
+            } else if (value instanceof Duration) {
+                propertiesValue.put(entry.getKey(), new ServiceBusDescribedType(ServiceBusConstants.DURATION_SYMBOL, convertToTickTime((Duration) value)));
+            }
+        }
+        return propertiesValue;
+    }
+
+    private Map<String, Object> convertToOriginType(Map<String, Object> propertiesValue) {
```

Comment:
Could you add a bit of documentation as to what this does? Also, I think this can be static.